src/clients/SpokePoolClient.ts

```diff
@@ -866,4 +869,19 @@ export class SpokePoolClient extends BaseAbstractClient {
 
     return deposit;
   }
+
+  /**
+   * Determines whether a deposit originates from a lite chain.
+   * @param deposit The deposit to evaluate.
+   * @returns True if the deposit originates from a lite chain, false otherwise. If the hub pool client is not defined,
+   *          this method will return false.
+   */
+  protected doesDepositOriginateFromLiteChain(deposit: DepositWithBlock): boolean {
+    return (
+      this.hubPoolClient?.configStoreClient.isChainLiteChainAtTimestamp(
+        deposit.originChainId,
+        deposit.quoteTimestamp
+      ) ?? false
```

Comment:
I'm not sure about this logic - it forces the deposit to be a lite deposit for eternity. Isn't the "lite chain" check relevant for the fill instead? We could in theory have a deposit with long expiry made on a lite chain, but the fill could occur after it is removed from the set of lite chains.

Reply:
This is something I discussed with Nick. I initially thought I would put the check on the current block time because this is when the fill is made.

I think there's tradeoffs to determining the lite chain inclusion at fill time or at deposit time - will discuss offline

Reply:
Resolved offline. Will move forward with this current direction